app/models/bookings/placement_request.rb

```diff
@@ -34,6 +34,23 @@ class PlacementRequest < ApplicationRecord
       dependent: :destroy
 
     scope :open, -> do
+      not_cancelled.merge unbooked
+    end
+
+    scope :unbooked, -> do
+      left_joins(:booking)
+        .where(bookings_bookings: { bookings_placement_request_id: nil })
+    end
+
+    scope :cancelled, -> do
+      left_joins(:candidate_cancellation, :school_cancellation)
+        .where <<~SQL
```

Comment:
Very minor point, but I'd favour the scope-merging approach here

Reply:
I _think_ I tried that approach originally but ran into an issue due to the relations being the same table